docker-registry-cleanup.py

```diff
@@ -125,20 +126,31 @@ def exit_with_error(message):
 
 		#Get the manifest json to check if its a manifest list
 		k.key = "%s/sha256/%s/%s/data" % (blob_dir, shasum[0:2], shasum)
-		manifest = json.loads(k.get_contents_as_string().decode())
+		try:
+			manifest = json.loads(k.get_contents_as_string().decode())
+		except OSError as e:
```

Comment:
I think this should just catch a general exception, as it's pointing to S3, so it uses a library and http behind the scenes, e.g. it would probably get a 404 or something like that if a file was missing on the S3 side of things.